web3/_utils/module_testing/shh_module.py

```diff
@@ -393,10 +393,7 @@ def test_shh_post_deprecated(self, web3: "Web3") -> None:
     #
     def test_shh_version(self, web3: "Web3") -> None:
         version = web3.geth.shh.version()
-        if '1.7' in web3.clientVersion:
-            assert version == '5.0'
-        else:
-            assert version == '6.0'
+        assert version == '6.0'
```

Comment:
FYI - latest version(s) of Geth removed Whisper from the distro, so when we bump to the latest version, maybe we want to look at dropping support for it in Web3.py.